.github/workflows/clean.yml

```diff
@@ -12,11 +12,12 @@ on: push
 jobs:
   delete-artifacts:
     name: Delete Artifacts
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Why the change to macos?

Reply:
Wasn't on purpose, seemed to change when workflow was regenerated.... I'll take a look.

Reply:
nbd, I see both ubuntu and macos in the CI. Bigger deal is CI failure.

Reply:
I corrected it by changing the order of the `githubWorkflowOSes` key.